UI/window-basic-main.cpp

```diff
@@ -1807,13 +2133,31 @@ void OBSBasic::CreateHotkeys()
 			"OBSBasic.Transition",
 			Str("Transition"), transition, this);
 	LoadHotkey(transitionHotkey, "OBSBasic.Transition");
+
+	takeScreenShotHotkey = obs_hotkey_register_frontend(
+		"OBSBasic.SaveMainOutputScreenShot",
+		Str("Basic.Main.SaveMainOutputScreenShot"),
+		[](void* data, obs_hotkey_id, obs_hotkey_t*, bool pressed)
+		{
+			OBSBasic &basic = *static_cast<OBSBasic*>(data);
+
+			if (pressed)
+			{
+				basic.SaveScreenshotHandler();
+			}
+		},
+		this);
+
```

Comment:
Code styling inconsistencies again.

To fix, you could do something like this:

```cpp
	takeScreenShotHotkey = obs_hotkey_register_frontend(
			"OBSBasic.SaveMainOutputScreenShot",
			Str("Basic.Main.SaveMainOutputScreenShot"),
			[] (void *data, obs_hotkey_id, obs_hotkey_t*,
					bool pressed)
			{
				OBSBasic &basic = *static_cast<OBSBasic*>(data);
                
				if (pressed) {
					basic.SaveScreenshotHandler();
				}
			},
			this);
```

Or if you want to make it a bit less squishy and clean looking:

```cpp
	auto screenshot_cb = [] (void *data, obs_hotkey_id, obs_hotkey_t*,
			bool pressed)
	{
		OBSBasic &basic = *static_cast<OBSBasic*>(data);

		if (pressed) {
			basic.SaveScreenshotHandler();
		}
	};

	takeScreenShotHotkey = obs_hotkey_register_frontend(
			"OBSBasic.SaveMainOutputScreenShot",
			Str("Basic.Main.SaveMainOutputScreenShot"),
			screenshot_cb,
			this);
```